src/components/API.js

```diff
@@ -0,0 +1,76 @@
+export default class Api {
```

Comment:
It’s better to move api class to utils directory, since it's not a component, but a utility class